react/features/toolbox/defaultToolbarButtons.js

```diff
@@ -153,7 +153,9 @@ const buttons: Object = {
         shortcutAttr: 'toggleDesktopSharingPopover',
         shortcutFunc() {
             JitsiMeetJS.analytics.sendEvent('shortcut.screen.toggled');
-            APP.conference.toggleScreenSharing();
+
+            // eslint-disable-next-line no-empty-function
+            APP.conference.toggleScreenSharing().catch(() => {});
```

Comment:
Note that there isn't new error introduced. The only change is that we were catching all the errors and not returning them which was resulting in resolved promise. But we need to receive the errors in the remote control module in order to determine whether the remote control session can start or not.

All the catch clauses are just preventing global unhandled rejected promises.